cmd/keytransparency-client/grpcc/grpc_client.go

```diff
@@ -139,7 +139,11 @@ func min(x, y int32) int32 {
 
 // ListHistory returns a list of profiles starting and ending at given epochs.
 // It also filters out all identical consecutive profiles.
+// Epochs start at 1.
 func (c *Client) ListHistory(ctx context.Context, userID, appID string, start, end int64, opts ...grpc.CallOption) (map[*trillian.SignedMapRoot][]byte, error) {
+	if start <= 0 {
```

Comment:
I added a test-case to document this isn't meant to be called with a start <0: https://github.com/google/keytransparency/pull/700/commits/c503ec6a33828321d1e566492cf9136b01fc3260#diff-f8b79d8d11b0c9af5cfa9f8d823cbb91L225